application/auth/models.py

```diff
@@ -82,6 +82,13 @@ def is_admin_user(self):
     def can(self, capability):
         return capability in self.capabilities
 
+    def can_access_measure(self, measure):
+        if self.user_type != TypeOfUser.DEPT_USER:
+            return True
+        else:
+            return self in measure.shared_with
+
+    # DEPRECATED: use `can_access_measure` method instead.
```

Comment:
This `can_access` method is only used in one place - by the `user_has_access` decorator, so rather than deprecating it can we just update that to use the new version and delete this one right away?

Reply:
@TheDoubleK looks like this is more complicated than it first appears. Maybe we note it as a bug/issue and tackle it separately?

Reply:
OK yep let's get this in.  But the access control is actually broken, so I'll write the ticket up now.

Reply:
👍